fuse_core/include/fuse_core/graph.h

```diff
@@ -316,6 +316,14 @@ class Graph
    */
   virtual void holdVariable(const UUID& variable_uuid, bool hold_constant = true) = 0;
 
+  /**
+   * @brief Check whether a variable is on hold or not
+   *
+   * @param[in] variable_uuid The variable to test
+   * @return True if the variable is on hold, false otherwise
+   */
+  virtual bool isVariableOnHold(const UUID& variable_uuid) const = 0;
```

Comment:
I don't really care, but would `isVariableHeld` be a better name?

Reply:
Yep, I thought about that. The only reason I ended up calling it `isVariableOnHold` is because the graph maintains a set of variables held in an attribute called `variables_on_hold_`.

@svwilliams Do you have any preference?

I'm happy to make the change.

Reply:
I have no opinion on the name either way. Fight amongst yourselves. 🙂

Reply:
lol

https://coinflip.com :thinking: :wink:

Reply:
@ayrton04 If you prefer `isVariableHeld`, simply let me know and I'll rename this method. No problem. Otherwise, I guess you can merge as is. :smiley:

Reply:
Oh, this can be merged as-is. I don’t really have strong feelings at all.